### PR TITLE
1.29.2 fix for legacy applications

### PR DESCRIPTION
**Pull request by jrushlow**
fix applied in https://github.com/symfony/maker-bundle/pull/882 which provides support for legacy Symfony 3.x applications using MakerBundle <1.29.x - Symfony 3.x support was dropped in MakerBundle starting with `v1.30.0`

- [x] release `1.29.2`

### CODE REVIEW

File: symfony/maker-bundle/CVE-2021-21424.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ branches:
         versions: ['>=1.28.0', '<1.29.0']
     1.29.x:
         time:     2021-05-12 08:00:00
```
Comment (stof) [inline symfony/maker-bundle/CVE-2021-21424.yaml:12]: this time will need to be updated once the bundle PR is merged

**Comment by weaverryan** [comment]
1.29.2 is tagged

**Comment by jrushlow** [comment]
https://github.com/symfony/maker-bundle/releases/tag/v1.29.2

File: symfony/maker-bundle/CVE-2021-21424.yaml (hunk 1 above)
Comment (jrushlow) [reply symfony/maker-bundle/CVE-2021-21424.yaml]: done

**Review by stof — APPROVED** [review]

**Pull request merged**